revisions.md

```diff
@@ -0,0 +1,60 @@
+# Odoo Revisions
+
+A Odoo revision is a snapshot of git references of the Odoo source and modules at a specific date.
+
+For each major release there are multiple revisions.
+
+## 13.0
```

Comment:
Is see the entry but no the actual script.

Reply:
The revisions were created manually, but this is a generated heading since there is a v13.0... revision. I will add the 13.0